lighthouse-core/config/default.json

```diff
@@ -106,6 +106,7 @@
     "description": "These audits validate the aspects of a Progressive Web App.",
```

Comment:
Descriptions support markdown. I'd prefer we just add a link here in the aggregation description rather than pipe a new property throughout the codebase. We already have a few existing places like `extendedInfo` and `description` text for help text like this. IMO, it's also better to have this at the top of the section rather than the bottom. It'll get missed down there.

"description": "These audits validate the aspects of a Progressive Web App. They're a subset of the [PWA Checklist](https://developers.google.com/web/progressive-web-apps/checklist)."